models/Recipe.js

```diff
@@ -36,7 +36,7 @@ Recipe.init({
     },
```

Comment:
Really simple fix, but I needed to add 0-9 to the regex function here so that instructions don't split on numbers used like this: "Then cook at 400 degrees for 4 hours"